src/GraphQL.php

```diff
@@ -151,6 +163,14 @@ public function queryAndReturnResult($query, ?array $variables = [], array $opts
         return GraphQLBase::executeQuery($schema, $query, $rootValue, $context, $variables, $operationName, $defaultFieldResolver);
     }
 
+    /**
+     * @return array<string>
+     */
+    public function executionMiddleware(): array
+    {
+        return config('graphql.execution_middleware', []);
```

Comment:
I think it's important we consider the schema name too (*), something like:
```php
return
    config("graphql.$schemaName.execution_middleware") ??
    config('graphql.execution_middleware') ??
    [];
```

(*) which isn't in this context yet, I know